### PR TITLE
all sorts of stuff

### PR DESCRIPTION
**Pull request by alekiponi**

### CODE REVIEW

File: gradle.properties
Comment (Traister101) [inline gradle.properties:1]: Why are we updating TFC?

File: src/main/java/mod/traister101/sacks/common/items/SNSItems.java
Comment (Traister101) [inline src/main/java/mod/traister101/sacks/common/items/SNSItems.java:1]: Wonky formatting?

File: resources/data.py
Comment (Traister101) [inline resources/data.py:1]: I'd rather the SACKS list just get removed/have things taken out than exceptions like this

**Review by Traister101 — CHANGES_REQUESTED** [review]

File: gradle.properties
Comment (alekiponi) [reply gradle.properties:1]: something was broken, I dont remember what

File: src/main/java/mod/traister101/sacks/common/items/SNSItems.java
Comment (alekiponi) [reply src/main/java/mod/traister101/sacks/common/items/SNSItems.java:1]: I didn't change the formatting at all

**Pull request merged**